src/SAF.DAS.ApprenticeCommitments.Web.UnitTests/Bindings/Web.cs

```diff
@@ -47,7 +52,7 @@ public void Initialise()
                 };
 
                 ActionResultHook = new Hook<IActionResult>();
-                Factory = new LocalWebApplicationFactory<ApplicationStartup>(_context, Config, ActionResultHook);
+                Factory = new LocalWebApplicationFactory<ApplicationStartup>(Config, ActionResultHook, _time);
```

Comment:
Would it have been better to keep the TestContext in extract timer Function from that. We are doing this above and as we add more classes like this, we will keep needing to do this.

Reply:
I tried having the timer in the `context` to start with, it didn't work as the context is continually recreated but the web `Factory` is only instantiated once.  It would not be able to use the `context` available to be as that context will be disposed of at the end of the first test.

It works this way round as the `_time` func is only created once, but the `_timeProvider` that's captured is updated.